routers/web/user/oauth.go

```diff
@@ -527,7 +527,7 @@ func GrantApplicationOAuth(ctx *context.Context) {
 		handleServerError(ctx, form.State, form.RedirectURI)
 		return
 	}
-	ctx.Redirect(redirect.String(), 302)
+	ctx.Redirect(redirect.String(), http.StatusSeeOther)
```

Comment:
You are sure about HTTP 303 on this? Where is `Location` header`?

Reply:
`http.Redirect` sets it.